packages/agent-toolkit/src/shared/tools.ts

```diff
@@ -39,114 +53,130 @@ export type Tool = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   parameters: z.ZodObject<any, any, any, any>;
   actions: Actions;
+  execute: (
```

Comment:
added this `execute` method so that you can grab all `tools` as an export and more easily build your own adapter with everything included.